examples/jsm/nodes/shadernode/ShaderNode.js

```diff
@@ -48,7 +48,7 @@ const shaderNodeHandler = {
 
 				const nodeElement = NodeElements.get( prop.slice( 0, prop.length - 'Assign'.length ) );
 
-				return ( ...params ) => nodeObj.assign( nodeElement( nodeObj, ...params ) );
+				return node.isStackNode ? ( ...params ) => nodeObj.assign( params[ 0 ], nodeElement( ...params ) ) : ( ...params ) => nodeObj.assign( nodeElement( nodeObj, ...params ) );
```

Comment:
Could we please also do this in the `if ( NodeElements.has( prop ) )` clause? Then we can remove `StackNode.loop` and `StackNode.assign`.

Reply:
(Fixed in https://github.com/mrdoob/three.js/pull/26821)